src/index.js

```diff
@@ -33,25 +38,34 @@ app.use(boom());
 bodyParser.urlencoded({ extended: true });
 app.use(bodyParser.json());
 app.use(express.urlencoded({ extended: true }));
-app.use('/swagger', swaggerUi.serve, swaggerUi.setup(swagger, { explorer: true }));
+app.use(
+  '/swagger',
+  swaggerUi.serve,
+  swaggerUi.setup(swagger, { explorer: true })
+);
 expressWinston.requestWhitelist.push('body');
 app.use(expressWinston.logger(loggerOptions));
 app.use(expressWinston.errorLogger(loggerOptions));
 
 // DB connect
-mongoose.connect(process.env.MONGO_URL);
+// mongoose.connect(process.env.MONGO_URL);
+mongoose.connect('mongodb://127.0.0.1:27017/test', { useNewUrlParser: true });
 
 // Routes
 app.use('/users', usersRouter);
 app.disable('etag');
 
-app.listen(process.env.PORT, (err) => {
-  if (err) {
-    console.log(err.message);
-  }
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
Our app must not care about tests and test environment

Reply:
Because when I execute the tests, I am getting this error for createUser():
listen EADDRINUSE: address already in use :::5000